app/src/main/java/com/sereno/serenoandroidapp/activities/WeatherInfoActivity.kt

```diff
@@ -40,6 +39,8 @@ class WeatherInfoActivity : AppCompatActivity() {
             .build()
 
         val inputCityNameText = findViewById<EditText>(R.id.inputCityNameText)
+        val outputWeatherIcon = findViewById<ImageView>(R.id.outputWeatherIcon)
+        val outputWeatherText = findViewById<TextView>(R.id.outputWeatherText)
```

Comment:
```suggestion
        val currentWeatherIcon = findViewById<ImageView>(R.id.currentWeatherIcon)
        val currrentWeatherText = findViewById<TextView>(R.id.currentWeatherText)
```
input の反対は output わかるけど、現在の天気とアイコンなので